admin/api_performance_advisor.go

```diff
@@ -488,10 +488,7 @@ func (a *PerformanceAdvisorApiService) listSlowQueriesExecute(r ListSlowQueriesA
 	if r.namespaces != nil {
 		t := *r.namespaces
 		if reflect.TypeOf(t).Kind() == reflect.Slice {
```

Comment:
Do we need this if?

Reply:
Good point. It is not needed for execution phase. It is because we do import reflect.
I will create workaround for that outside that function